SRC/PORT.CPP

```diff
@@ -21,20 +22,21 @@ static mach_timebase_info_data_t timer_timebase_info;
 static uint64_t timer_zero = 0;
 #endif
 
-
-static void tk_port_screen_test();
-
 static Uint32 tk_port_update_framecount( Uint32 interval, void *param )
 {
 	framecount++;
 	// Return the next interval to be used. This technically allows
 	// for dynamic speed changes during the game by mutating the `target_frames`
 	// global...
-	return target_frames;
+	return 1000 / target_frames;
 }
 
 int tk_port_init(  )
 {
+	tk_port_debug =  strtoul(getenv("TK_PORT_DEBUG"), NULL, 10);
```

Comment:
This segfaults if `TK_PORT_DEBUG` isn't defined.

Reply:
Ah, my bad. I would've figured `strtoul` would handle NULL input as zero, but... hey, it's C.